runtime/testy/studentCode.py

```diff
@@ -8,16 +8,53 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  print("setup")
+  devices = Robot.getValue("hibike", "devices")
```

Comment:
This needs to be changed to `Robot._getSMValue` (along with the other calls below) in order to work on `master`.